index.js

```diff
@@ -1,416 +1,416 @@
 
-const isEven = require('not-odd');
```

Comment:
my editor suggested I make this change to make it an ecmascript module. ftfw btw